RSS Reader/RSS Reader/Categories/UIColor/UIColor+RSSRColor.m

```diff
@@ -9,12 +9,12 @@
 
 @implementation UIColor (RSSRColor)
 
-+ (UIColor *)RSSRWhite {
-    return [UIColor colorNamed:@"White color"];
++ (UIColor *)RSSRBackgroundColor {
```

Comment:
Good. But it would be even better to store the assets inside the assets catalogue

Reply:
I have this color set in the project assets and I use properties in the category just to easily access it
So I’m not quite sure I understand you. Please, could you describe it in more detail

Reply:
Good that you already did this